public/components/schemas/TransactionBody.yml

```diff
@@ -0,0 +1,172 @@
+type: object
+properties:
+  transactionId:
+    type: string
+    format: ObjectId
+    externalDocs:
+      description: Unique identifier
+      url: '/#section/Introduction/Unique-Identifier'
+    description: ID of the transaction
+    example: 62b9437318fe4517089b2833
+  amount:
```

Comment:
All these fields seems to be copy pasted from another file. It would be nice if we could reference the other file instead. Using `allOf` kind of syntax. So that if we want to update the documentation for `amount` for example we only need to do it at one place.

It might make sense that this file should become the source of truth and that instead the other schemas should import this schema.